i18n/id/docusaurus-plugin-content-docs/current/Community-Checking/adding-questions.md

```diff
@@ -0,0 +1,137 @@
+
+Ketik pertanyaan Anda di kotak **Pertanyaan**. Jika Anda ingin menambahkan versi audio dari pertanyaan tersebut, klik tombol **Rekam**. Anda mungkin akan diminta untuk memberikan izin untuk mengakses mikrofon Anda. Setelah Anda selesai merekam, klik **Hentikan Perekaman**. Kemudian, Anda dapat memutar rekaman untuk mendengar hasilnya. Klik tombol **Coba Lagi** jika Anda tidak puas dengan hasilnya. Anda juga dapat mengunggah file audio alih-alih merekam, jika Anda mau.
+
+Klik **Simpan** dan pertanyaan Anda akan ditambahkan ke daftar pertanyaan pemeriksaan komunitas.
+
+![](./1916935940.png)
+
+### Mengimpor pertanyaan dari spreadsheet {#1850d745ac9e8085960dd88b648f0c7a}
+
+Membuat pertanyaan satu per satu bisa jadi membosankan, jadi kami juga memungkinkan untuk mengimpor pertanyaan dari spreadsheet. Anda dapat menggunakan program seperti Microsoft Excel, Google Spreadsheet, atau LibreOffice Calc untuk membuat atau menyunting daftar pertanyaan. Jika Anda mencari satu set pertanyaan yang telah dibuat sebelumnya, pertimbangkanlah [unfoldingWord® Translation Questions] (https://git.door43.org/unfoldingWord/en_tq). Jika Anda membuka halaman yang ditautkan dan mengklik salah satu file, Anda dapat mengunduhnya, membukanya sebagai spreadsheet, dan menyunting daftar pertanyaan.
+
+Berikut adalah contoh bagaimana baris-baris dalam spreadsheet Anda seharusnya terlihat agar dapat diimpor ke Scripture Forge:
+
+| Referensi                 | Pertanyaan                                              |
+| ------------------------- | ------------------------------------------------------- |
+| Ibr 1:1   | Apa yang dimaksud dengan "nenek moyang kita"?           |
+| Ibr 1:2   | Apa artinya bahwa Anak adalah "pewaris segala sesuatu"? |
+| Ibr 1:2-3 | Siapakah "Anak" yang dibicarakan dalam ayat-ayat ini?   |
+
+Spreadsheet dapat memiliki lebih banyak kolom, tetapi semua kolom kecuali kolom "ayat referensi" dan "pertanyaan" tidak akan digunakan. File TSV yang diterbitkan oleh unfoldingWord memiliki format yang sedikit berbeda, tetapi dapat diimpor ke Scripture Forge juga.
+
+:::note
+
+Referensi harus memiliki nama kitab yang disingkat dengan cara yang sama seperti Paratext menyingkat nama buku. Sebagai contoh, **Ibr 1:1** dapat digunakan, tetapi **Ibrani 1:1** tidak dapat digunakan.
+
+:::
+
+Jika Anda membutuhkan bantuan untuk memasukkan spreadsheet Anda ke dalam format yang tepat untuk diimpor ke Scripture Forge, kami akan dengan senang hati membantu Anda. Cukup kirimkan email ke [help@scriptureforge.org](mailto:help@scriptureforge.org).
+
+1. Simpan spreadsheet Anda menjadi file CSV.
+
+    ::: catatan
```

Comment:
People should not be translating these from "note" (or whatever it was) to this, because then it won't render as a note or whatever.

Reply:
I'm also seeing `:::Anmerkung` and `:::remarque`